Milestone 1: Portfolio setup

dewslyse opened the pull request:
### Portfolio: setup and mobile version skeleton

In this milestone I:
- set up a new repository and prepare it for development using best practices
- built the first two sections of the mobile website

dewslyse committed:
Configure linters

dewslyse committed:
Add index.html file

dewslyse committed:
Add stylesheet to project

dewslyse committed:
Add .gitignore

dewslyse committed:
Add via upload

dewslyse committed:
Add toolbar and headline sections

dewslyse committed:
Add via upload

dewslyse committed:
Update README.md

micheaol commented on index.html:
```diff
@@ -0,0 +1,53 @@
+
+<body>
+    <main class="wrapper">
+        <header class="header">
```
The `header tag` should only be wrapped around the toolbar (headline section not inclusive). Please remove the `headline section` from the `header`.

You should have something like this:

` <header></header>`
`<section></section>`

micheaol commented on index.html:
```diff
@@ -0,0 +1,53 @@
+        <header class="header">
+            <div class="toolbar">
+                <a class="logo" href="#" aria-label="Homepage">>DS_</a>
+                <img src="./images/menu-btn.svg" alt="">
```
Please ensure that the hamburger button has a `nav` tag according to the requirement.

micheaol commented on index.html:
```diff
@@ -0,0 +1,53 @@
+                <img src="./images/menu-btn.svg" alt="">
+            </div>
+
+            <div class="headline">
+                <h1 class="intro">I’m Selase 👋 Glad to see you!</h1>
+                <p class="description">
+                    I’m a software developer! I can help you build a product , feature or website. Look through some of
+                    my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+                    to contact me.
+                </p>
```
Please ensure that the container of this section is a section tag according to the requirement.

micheaol commented on index.html:
```diff
@@ -0,0 +1,53 @@
+                    to contact me.
+                </p>
+                <p class="social-media">Let's connect</p>
+                <div class="socials">
+                    <a class="social-icon" href="#" aria-label="twitter">
+                        <img src="./images/twitter.svg" alt="">
+                    </a>
+                    <a class="social-icon" href="#" aria-label="Linkedin">
+                        <img src="./images/Linkedin.svg" alt="">
+                    </a>
+                    <a class="social-icon" href="#" aria-label="medium">
+                        <img src="./images/medium.svg" alt="">
+                    </a>
+                    <a class="social-icon" href="#" aria-label="Github">
+                        <img src="./images/github.svg" alt="">
+                    </a>
+                    <a class="social-icon" href="#" aria-label="AngelList">
+                        <img src="./images/AngelList.svg" alt="">
+                    </a>
```
Please ensure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)

micheaol commented on style.scss:
```diff
@@ -0,0 +1,79 @@
+  text-decoration: none;
+}
+
+.headline {
```
Please ensure that the content of `headline section` is aligned with the flexbox

micheaol reviewed (changes requested):
Hi @dewslyse , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️

- _Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

dewslyse committed:
Align headline section with flexbox

dewslyse committed:
Resolve issues from pull request

- Remove headline section from header
- Wrap menu button in nav tag
- Change headline div to section tag
- Use <ul> for social icons

micheaol reviewed (approved):
# STATUS: APPROVED :green_circle: :clap: 

Hi @dewslyse , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

![well-1](https://user-images.githubusercontent.com/64173189/128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
